docs/1.md

```diff
@@ -1459,6 +1459,7 @@ t = sorted(s)  # s sin cambios, t tiene los valores ordenados
 ### 1.5.6: Listas y matemáticas
 
 *Caution: Lists were not designed for math operations.*
+*Cuidado: Las listas no son designados para operaciones matemáticas*
```

Comment:
*not designed* tiene que ver con diseño en vez de designación; las listas no fueron diseñadas para eso